### PR TITLE
docs: Document some non-obvious features.

### PR DESCRIPTION
**Pull request by vpetrovykh**
Document how to run EdgeQL scripts.

Document the fact that exclusive constraints automatically create an
index on the constraint expression.

### CODE REVIEW

File: docs/datamodel/constraints.rst [hunk 1]
```diff
@@ -164,6 +164,11 @@ The standard library defines the following constraints:
             }
         }
 
+    This constraint also has an additional effect of creating an
+    implicit :ref:`index <ref_datamodel_indexes>` on the link or
+    property. This means that in the above example there's no need to
+    add explicit indexes for the ``name`` property or the ``owns``
```
Comment (elprans) [inline docs/datamodel/constraints.rst]: Indexes on links are not necessary, they're always created implicitly.

Comment (vpetrovykh) [reply docs/datamodel/constraints.rst]: OK, I've updated the docs to reflect that,

File: docs/datamodel/constraints.rst [hunk 2]
```diff
@@ -164,6 +164,10 @@ The standard library defines the following constraints:
             }
         }
 
+    This constraint also has an additional effect of creating an
```
Comment (1st1) [inline docs/datamodel/constraints.rst]: I'd put this in a NOTE block to highlight.

**Comment by 1st1** [comment]
@vpetrovykh Please merge if ready.

**Pull request merged**